packages/paste-theme/src/themeProvider.tsx

```diff
@@ -22,10 +22,10 @@ function getProviderThemeProps(theme: ThemeVariants, customBreakpoints?: string[
         ...SendGridTheme,
         breakpoints: customBreakpoints || SendGridTheme.breakpoints,
       };
-    case ThemeVariants.CONSOLE:
+    case DeprecatedThemeVariants.CONSOLE:
       return {
-        ...ConsoleTheme,
-        breakpoints: customBreakpoints || ConsoleTheme.breakpoints,
+        ...DefaultTheme,
+        breakpoints: customBreakpoints || DefaultTheme.breakpoints,
       };
```

Comment:
If you omit these lines, the case will fall into the DEFAULT block and _just work_